src/main/resources/lang/default.lang

```diff
@@ -2418,6 +2418,32 @@ experience cooldown change reasons:
 	plugin: plugin
 	pickup_orb: orb pickup, pickup orb
 
+# -- Recipe Categories --
+recipe categories:
+	crafting_building: crafting building, craftingcategory building, crafting category building
+	crafting_equipment: crafting equipment, craftingcategory equipment, crafting category equipment
+	crafting_misc: crafting misc, craftingcategory misc, crafting category misc
+	crafting_redstone: crafting redstone, craftingcategory redstone, crafting category redstone
+	cooking_blocks: cooking blocks, cookingcategory blocks, cooking category blocks
+	cooking_food: cooking food, cookingcategory food, cooking category food
+	cooking_misc: cooking misc, cookingcategory misc, cooking category misc
```

Comment:
a lot of these are missing spaces between words, misc should have miscellaneous as an option, and the ordering should be `misc cooking category`, not `cooking category misc`